packages/phone-number-privacy/signer/src/signing/query-quota.ts

```diff
@@ -209,7 +219,7 @@ async function getQueryQuota(logger: Logger, account: string, hashedPhoneNumber?
     quota: 0,
   })
   logger.debug({ account }, 'Account is not verified and does not meet min balance')
-
+  getBalancesMeter()
```

Comment:
rather than putting this in two places why not just have it tight around the calls that get the balances? The rest of this code doesn't seem like it would produce much latency